### PR TITLE
Fixup/nick/car view

### PR DESCRIPTION
**Pull request by nickknapton12**
Car svg added to car modal with support for dark and light mode.

Battery and Energy Tabs added by Thomas, fixed the sizing of tabs popping up.

Fixed Linter mistake causing last PR to fail tests :(

### CODE REVIEW

**Review by rjb75 — APPROVED** [review]
Thanks for finishing up the car stuff @nickknapton12. Once we get another review we can merge these changes in. If you could hold off on automerge just so I can make sure it all goes in okay.

File: src/pages/CarModals/Battery.tsx
Comment (RatikKapoor) [inline src/pages/CarModals/Battery.tsx:21]: Comments are always 😍

Comment (RatikKapoor) [inline src/pages/CarModals/Battery.tsx:21]: Comments are 😍

**Review by RatikKapoor — APPROVED** [review]
🙌🏽 looks good! My only suggestion would be to add a drop shadow or something to the second modal since it's quite difficult to distinguish from the first. This could be handled in another PR or this one depending on which way you want to go. Approving for now

**Comment by RatikKapoor** [comment]
CC @rjb75 for the call on drop shadow now vs later

**Comment by rjb75** [comment]
> CC @rjb75 for the call on drop shadow now vs later

We can just leave it for now. We will probably standardize a few things in the next week or two.

**Pull request merged**
